devlib/module/cgroups.py

```diff
@@ -297,6 +292,14 @@ def add_tasks(self, tasks):
     def add_proc(self, pid):
         self.target.write_value(self.procs_file, pid, verify=False)
 
+    def _attr_path(self, entry):
```

Comment:
Rename to `build_attr_path` and get rid of the subsequent comment.

- Method names should generally be verbs
- Since this is invoked outside the class implementation, it's not an internal method, and should not start with a `_`.